var/spack/repos/builtin/packages/gperf/package.py

```diff
@@ -20,4 +20,6 @@ class Gperf(AutotoolsPackage, GNUMirrorPackage):
     version("3.1", sha256="588546b945bba4b70b6a3a616e80b4ab466e3f33024a352fc2198112cdbb3ae2")
     version("3.0.4", sha256="767112a204407e62dbc3106647cf839ed544f3cf5d0f0523aaa2508623aad63e")
 
+    patch("register.patch")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from (if applicable)

? That is basically to help readers when they go through the recipe.